lms/djangoapps/instructor/views/legacy.py

```diff
@@ -734,10 +749,21 @@ def get_student_grade_summary_data(
                         if get_score_max is True:
                             add_grade(score.section, score.earned, score.possible)
                         else:
-                            add_grade(score.section, score.earned)
+                            add_grade(score.section, (score.earned / score.possible), 1)
                 else:
+                    category_cnts = Counter()
                     for grade_item in gradeset['section_breakdown']:
-                        add_grade(grade_item['label'], grade_item['percent'])
+                        category = grade_item['category']
+                        try:
+                            earned = gradeset['totaled_scores'][category][category_cnts[category]].earned
+                            possible = gradeset['totaled_scores'][category][category_cnts[category]].possible
+                            if get_score_max is True:
+                                add_grade(grade_item['label'], earned, possible=possible)
+                            else:
+                                add_grade(grade_item['label'], grade_item['percent'], possible=1)
```

Comment:
@pdpinch @pwilkins 

- Reverted code for normalize  calculation as previous code, shared in meeting https://github.com/mitocw/edx-platform/blame/df5b2dadda636184116fa394105ea37e3ab3b867/lms/djangoapps/instructor/views/legacy.py#L731

-  Added ``possible=1`` because code above (https://github.com/mitocw/edx-platform/pull/190/files#diff-5dd41ae768fed10fe37df53a1d9e78c8R268) is expecting a tuple (earned, possible).

- If this is an issue the i need to modify code to release data without possible in 
https://github.com/mitocw/edx-platform/pull/190/files#diff-5dd41ae768fed10fe37df53a1d9e78c8R268 ??